instrumentation/runtime-metrics/runtime-metrics-java8/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetricsjava8/Java8RuntimeMetricsInstaller.java

```diff
@@ -3,34 +3,34 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.javaagent.runtimemetrics;
+package io.opentelemetry.instrumentation.javaagent.runtimemetricsjava8;
 
 import com.google.auto.service.AutoService;
 import io.opentelemetry.api.GlobalOpenTelemetry;
 import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.instrumentation.runtimemetrics.BufferPools;
-import io.opentelemetry.instrumentation.runtimemetrics.Classes;
-import io.opentelemetry.instrumentation.runtimemetrics.Cpu;
-import io.opentelemetry.instrumentation.runtimemetrics.GarbageCollector;
-import io.opentelemetry.instrumentation.runtimemetrics.MemoryPools;
-import io.opentelemetry.instrumentation.runtimemetrics.Threads;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.BufferPools;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.Classes;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.Cpu;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.GarbageCollector;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.MemoryPools;
+import io.opentelemetry.instrumentation.runtimemetricsjava8.Threads;
 import io.opentelemetry.javaagent.extension.AgentListener;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
 
 /** An {@link AgentListener} that enables runtime metrics during agent startup. */
 @AutoService(AgentListener.class)
-public class RuntimeMetricsInstaller implements AgentListener {
+public class Java8RuntimeMetricsInstaller implements AgentListener {
 
   @Override
   public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredSdk) {
     ConfigProperties config = autoConfiguredSdk.getConfig();
 
     boolean defaultEnabled = config.getBoolean("otel.instrumentation.common.default-enabled", true);
-    if (!config.getBoolean("otel.instrumentation.runtime-metrics.enabled", defaultEnabled)) {
+    if (!config.getBoolean("otel.instrumentation.runtime-metrics-java8.enabled", defaultEnabled)
```

Comment:
I think we might want to leave the old name: whether it is java 8 or 17, this is still the same feature.
```suggestion
    if (!config.getBoolean("otel.instrumentation.runtime-metrics.enabled", defaultEnabled)
```

Reply:
ok I've changed it back to the old name now